dcm4che-tool/dcm4che-tool-jpg2dcm/README.md

```diff
@@ -1,32 +1,34 @@
     usage: jpg2dcm [Options] <jpgfile> <dcmfile>
-    Encapsulate JPEG Image into DICOM Object.
+
+    Supports conversion of pixel data (with extensions jpg, jpeg, mpg, mpeg, mpg2) to DICOM.
+    User may choose to specify metadata via command line (using -m option) or
```

Comment:
Metadata can also be specified as XML file AND as command line arguments. Therefore I would write "... via command line (using -m option) and a file (using -f option) ...".

Minor language suggestion: Avoid the word "User" to describe to tool, but instead start the sentences like this: "Metadata can be specified via command line ..." / "The charset can also be specified ...".